### PR TITLE
Base home screen

### PR DESCRIPTION
**Pull request by NovitaGuok**
Only the boilerplate. Salah branching juga aku Kak, ketinggalan developnya wkwkwkw ntr merge anyway. For the next ada develop

### CODE REVIEW

File: app/src/main/java/com/example/ihonboarding/components/HomeHeader.kt [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.ihonboarding.R
+import com.example.ihonboarding.ui.theme.IHOnboardingTheme
+
+class HomeHeader : ComponentActivity() {
```
Comment (Admokonugroho) [inline app/src/main/java/com/example/ihonboarding/components/HomeHeader.kt:28]: I see a lot of components extend ComponentActivity better we create one activity and tree component, just suggestion [component semples](https://github.com/android/compose-samples/blob/main/JetNews/app/src/main/java/com/example/jetnews/ui/MainActivity.kt)

File: app/src/main/java/com/example/ihonboarding/components/HomeHeader.kt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            IHOnboardingTheme {
+                // A surface container using the 'background' color from the theme
+                Surface(color = MaterialTheme.colors.background) {
+                    Profile("Novita", "It's never too old to learn", "https://www.linkedin.com/in/novitaguok/")
```
Comment (Admokonugroho) [inline app/src/main/java/com/example/ihonboarding/components/HomeHeader.kt:35]: let's we put on strings xml

File: app/src/main/java/com/example/ihonboarding/components/HomeHeader.kt [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            }
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun DefaultPreview6() {
+    IHOnboardingTheme {
+        Profile("Novita", "It's never too old to learn", "https://www.linkedin.com/in/novitaguok/")
```
Comment (Admokonugroho) [inline app/src/main/java/com/example/ihonboarding/components/HomeHeader.kt:95]: for preview, it's okay hardcoded.

**Review by Admokonugroho — CHANGES_REQUESTED** [review]

**Review by elfatah — CHANGES_REQUESTED** [review]
Please put color in constant somewhere to make it reusable

**Pull request merged**